### PR TITLE
Added OCS CephMonLowNumber alert

### PR DESCRIPTION
**Pull request by jmolmo**
Fixes: 2257634
https://bugzilla.redhat.com/show_bug.cgi?id=2257634

### CODE REVIEW

File: alerts/openshift-container-storage-operator/CephMonLowNumber.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+## Mitigation
+
+If increasing the number of monitors to 5 is not a right option (for any cause),
+the alert can be silenced.
+
+If to increase the number of monitors is an acceptable proposal, then execute
+the following command to do that:
+
+```bash
+    oc scale deployment rook-ceph-mon --replicas=5 -n openshift-storage
```
Comment (Nikhil-Ladha) [inline alerts/openshift-container-storage-operator/CephMonLowNumber.md:45]: The right thing to do via CLI is setting the `monCount` to 5 in `StorageCluster` CR. i.e,
` oc patch storageclusters.ocs.openshift.io ocs-storagecluster -n openshift-storage --type merge --patch '{"spec": {"managedResources": [{"cephCluster": {"monCount" : 5}}]}}'`

Comment (Nikhil-Ladha) [reply alerts/openshift-container-storage-operator/CephMonLowNumber.md:45]: Give me sometime to confirm this command, please don't merge till then.

Comment (Nikhil-Ladha) [reply alerts/openshift-container-storage-operator/CephMonLowNumber.md:45]: Updated the command, it looks good now.

Comment (jmolmo) [reply alerts/openshift-container-storage-operator/CephMonLowNumber.md]: @Nikhil-Ladha , Thank you very much for the review and for the "fix"! Updated with your feedback.

**Review by Nikhil-Ladha — APPROVED** [review]
lgtm

**Review by umangachapagain — APPROVED** [review]

**Pull request merged**